libs/@guardian/eslint-config/.eslintrc.js

```diff
@@ -75,6 +75,7 @@ module.exports = {
 		],
 
 		// prevent circular dependencies
-		'import/no-cycle': 2,
+		// See: https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-cycle.md#ignoreexternal
```

Comment:
JSDoc maybe?

```suggestion
		/** @see https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-cycle.md#ignoreexternal */
```

Reply:
what will this change?

Reply:
I've dropped this change to be consistent with the rest of the file.